minio-mixin/.lint

```diff
@@ -0,0 +1,5 @@
+exclusions:
+  panel-units-rule:
+    reason: "The dashboard already has units but it is using older panel versions for which this rule does not work correctly"
+  alert-name-camelcase:
+    reason: "Rule incorrectly giving warning while alert names are already in camelCase"
```

Comment:
Should specify that the alert rule is due to having abbreviations/acronyms in the title, as info for future maintainers

Reply:
I have capitalized the initial letters and this rule is not giving an issue more so removed the lint exclusion.